app/src/main/java/org/mifos/selfserviceapp/ui/fragments/SavingsMakeTransferFragment.java

```diff
@@ -114,49 +118,59 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
     @OnClick(R.id.btn_review_transfer)
     void reviewTransfer() {
-        if (etAmount.getText().toString().equals("")) {
-            showToaster(getString(R.string.enter_amount));
-            return;
+        if (isConnectedToInternet()) {
```

Comment:
Use utils/Network.class for checking internet connection and make changes accordingly.